pelican/paginator.py

```diff
@@ -28,20 +28,30 @@ def __init__(self, name, object_list, settings):
         if settings.get('DEFAULT_PAGINATION'):
             self.per_page = settings.get('DEFAULT_PAGINATION')
             self.orphans = settings.get('DEFAULT_ORPHANS')
+            self.pagination_fixed = settings.get('PAGINATION_FIXED')
```

Comment:
This attribute name be updated appropriately if the setting name gets changed to `FIXED_PAGINATION`.